<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuDifficulty">
    <property name="title">
     <string>Difficulty</string>
    </property>
    <addaction name="action_Beginner_8x8_10_mines"/>
    <addaction name="actionIntermediate"/>
    <addaction name="actionExpert_16x30_99_mines"/>
    <addaction name="action_Wrapped"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuDifficulty"/>
  </widget>
  <action name="actionNewGame">
   <property name="text">
    <string>&amp;New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDifficulty">
   <property name="text">
    <string>Difficulty</string>
   </property>
  </action>
  <action name="action_Beginner_8x8_10_mines">
   <property name="text">
    <string>&amp;Beginner (8x8, 10 mines)</string>
   </property>
  </action>
  <action name="actionIntermediate">
   <property name="text">
    <string>&amp;Intermediate (16x16, 40 mines)</string>
   </property>
  </action>
  <action name="actionExpert_16x30_99_mines">
   <property name="text">
    <string>&amp;Expert (16x30, 99 mines)</string>
   </property>
  </action>
  <action name="action_Yes">
   <property name="text">
    <string>&amp;Yes</string>
   </property>
  </action>
  <action name="action_No">
   <property name="text">
    <string>&amp;No</string>
   </property>
  </action>
  <action name="action_Wrapped">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wrapped</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
